Source/WebCore/platform/graphics/FontCascade.cpp

```diff
@@ -1254,10 +1254,20 @@ bool FontCascade::canReceiveTextEmphasis(char32_t c)
         return false;
 
     // Additional word-separator characters listed in CSS Text Level 3 Editor's Draft 3 November 2010.
+    // https://www.w3.org/TR/css-text-3/#word-separator
     if (c == ethiopicWordspace || c == aegeanWordSeparatorLine || c == aegeanWordSeparatorDot
         || c == ugariticWordDivider || c == tibetanMarkIntersyllabicTsheg || c == tibetanMarkDelimiterTshegBstar)
         return false;
 
+    // Punctuations
```

Comment:
"Punctuations" is poor grammar. Should be "punctuation" or "punctuation characters". But also that comment doesn't explain the exceptions list and doesn't seem super helpful. I think we either should omit the comment or write a better one. The list of exceptions comes from the specification so we might want a link.